features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -120,6 +120,10 @@
             "help": "default network name for wisun network",
             "value": "\"Wi-SUN Network\""
         },
+        "wisun-network-size": {
+            "help": "Supported network size for wisun network: NETWORK_SIZE_SMALL, NETWORK_SIZE_MEDIUM, NETWORK_SIZE_LARGE",
```

Comment:
Where are these defines coming are these from nanostack? 

The size is 100 of devices in network.
I think it should be mentioned those defines are the limit of our internal concepts sizes for the network.

Reply:
could you change this to
"Expected amount of devices in the network as 100s of devices. with possible pre defined constants NETWORK_SIZE_SMALL, NETWORK_SIZE_MEDIUM, NETWORK_SIZE_LARGE, NETWORK_SIZE_XLARGE. if set to 0 Wi-SUN Certification configuration values are used."

Reply:
Updated as requested.